app/graph/types/team_type.rb

```diff
@@ -295,4 +295,14 @@ def tipline_messages(uid:)
   def articles(article_type:)
     object.explainers if article_type == 'explainer'
   end
+
+  field :api_key, ApiKeyType, null: true do
+    argument :dbid, GraphQL::Types::Int, required: true
+  end
+
+  def api_key(dbid:)
+    object.get_api_key(dbid)
+  end
+
+  field :api_keys, ApiKeyType.connection_type, null: true
```

Comment:
Since this is sensitive information, please implement a custom `resolve` for this connection, which checks if `ability.can?(:read, api_key)` for each API key returned (it will break pagination but it's OK - there is a limit on the number of API keys a workspace can create).